src/pages/workspaces/workspace/Data.js

```diff
@@ -509,7 +509,8 @@ class EntitiesContent extends Component {
         workspace,
         selectedEntities: _.keys(selectedEntities), selectedDataType: entityKey, runningSubmissionsCount
       }),
-      showIgvSelector && h(IGVFileSelector, {
+      h(IGVFileSelector, {
```

Comment:
Another thing: since this now stays mounted, it will remember the selection state through close/reopen unless we explicitly clear it. We'll probably want a `componentDidUpdate` that checks for a close -> open transition and reinitializes the selection state.

Reply:
I updated the file selector to clear the selections on dismiss